web/source/osk/activeLayout.ts

```diff
@@ -55,24 +88,39 @@ namespace com.keyman.osk {
         keys[j]['widthpc']=keyPercent;
```

Comment:
I'm confused as to why `width` and `pad` are stored as strings, and must be parsed as ints here.  Can I get a clarification?

Reply:
https://github.com/keymanapp/keyman/issues/119

In a single word:  legacy.